README.md

```diff
@@ -317,11 +317,13 @@ Once your import has completed with error, refresh the derived data views and
 confirm that nothing breaks:
 
 ```
-docker-compose run --rm app python manage.py make_materialized_views --recreate
 docker-compose run --rm app ./manage.py rebuild_index --noinput
 docker-compose run --rm app ./manage.py update_composition_index --recreate
 ```
 
+#### data_archive config
+If you need to test the `data_archive` make recipe, `cp .env.example .env` and add your AWS access tokens. These tokens must have the correct IAM policy for accessing the S3 bucket. Run  `docker-compose run --rm app make data_archive` so you can upload the zip archive to S3. Once this is done, the "download" link on the `~/download` page should work.
```

Comment:
Good question, I can't find the comment where I asked about your opinion on how we should set the different environments for targeting the appropriate S3 bucket @hancush ?

We have three different environments where  this could run:
1. locally
2. staging
3. production

So we need at least two S3 buckets (staging and production). Should we set an env variable for each environment? And if so, where/how is the best way do we do that? I can think of a couple but I'm curious to hear your preference.

Reply:
i setup config files similar to the local_settings.py where it copies the files in the `after_install.sh` script. i thought about adding the variable to the settings file, but decided against that because that would be declaring that variable in two places which could be error prone. i thought the json file was the easiest to use in make.

here's the commit: https://github.com/security-force-monitor/sfm-cms/pull/812/commits/1a7e1d0ddb5d80eaf1f881bd4c5b31028b60ce5b. note that i changed `sfm` to `wwic` after this commit.

after adding the .env.s3 file from your other comment, i realized we could do something like that but for these configs. i think we should do a video chat if we're gonna refine our approach.